internal_packages/keybase/spec/encrypt-button-spec.cjsx

```diff
@@ -0,0 +1,143 @@
+{React, ReactDOM, ReactTestUtils, DraftStore, Message} = require 'nylas-exports'
+pgp = require 'kbpgp'
+
+EncryptMessageButton = require '../lib/encrypt-button'
+PGPKeyStore = require '../lib/pgp-key-store'
+
+describe "EncryptMessageButton", ->
+  beforeEach ->
+    key = """-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
It'd be cool to move these test keys to a `fixtures` folder inside the spec folder for the plugin, and then load them in the specs via `key = fs.readFileSync('./fixtures/text-private-key.txt')`